libs/pong/get.js

```diff
@@ -1,141 +1,84 @@
 import he from "he";
 import anonymousIpByCC from "./cc2ip.js";
-import { fallbackHandler } from "./fallback.js";
 
 const PLACEMENTS = {
   side: 369,
   top: 585,
+  hpMain: 3214,
+  hpFooter: 2327,
 };
 
 // Allow list for client sent keywords.
 const ALLOWED_KEYWORDS = [];
 
-export function createPongGetHandler(client, coder, env) {
-  const { CARBON_ZONE_KEY, FALLBACK_ENABLED } = env;
-  return async (body, countryCode, userAgent) => {
+export function createPongGetHandler(client, coder) {
+  return async (body, countryCode = "us") => {
```

Comment:
I believe what we pass as `countryCode` defaults to `"US"` in the cloud function - should this also be uppercase?

Reply:
That should not be in there. That was for testing 🙈